RecoLocalTracker/SiPixelRecHits/interface/pixelCPEforGPU.h

```diff
@@ -208,31 +210,37 @@ namespace pixelCPEforGPU {
   }
 
   constexpr inline
-  void error(CommonParams const & __restrict__ comParams, DetParams const & __restrict__ detParams, ClusParams & cp, uint32_t ic) {
+  void errorOld(CommonParams const & __restrict__ comParams, DetParams const & __restrict__ detParams, ClusParams & cp, uint32_t ic) {
```

Comment:
is `errorOld(...)` used anywhere ?

Reply:
it can be used, I do not want to add config in cuda at this point
in principle there are already 3 different error source:
the parametrizaion by size (errorOld)
loading typical values (by det) from DB for normal 1pix, 1bigpix
as above but hardwired
one could try to combine and get a parametrization by size (as angle estimate) from the DB
(TBD)

Reply:
Yes, using names that convey where the errors come from seems a good idea.